src/dmd/compiler.d

```diff
@@ -12,7 +12,59 @@
 
 module dmd.compiler;
 
+///
 struct Compiler
 {
-    const(char)* vendor; // Compiler backend name
+    /// Compiler backend name
+    const(char)* vendor;
 }
+
+/**
+ * Initializes the dmd lexer and all required components in order
+ *
+ * It is templated (so D only) to prevent
+ *  errors on local imports to things that may not exist.
+ *
+ * See_Also:
+ *     dmd_lexer_deinit
+ */
+void dmd_lexer_init()()
```

Comment:
Nit: why underscores? [DStyle](https://dlang.org/dstyle.html#naming_conventions) is camelCase